examples/email-auth-local-storage/README.md

```diff
@@ -65,3 +67,30 @@ $ pnpm run dev
 ```
 
 This command will run a NextJS app on port 3000. If you navigate to http://localhost:3000 in your browser, you can follow the prompts to start an email auth.
+
+### 4/ Stackblitz Example
+
+You will need to create a Stackblitz account to run the following example
+
+Example Link: https://stackblitz.com/edit/stackblitz-starters-pyyw59
+
+#### Prerequisites
+
+To use the example you need the following
```

Comment:
Let's make sure to include these instructions in the stackblitz repo itself

Reply:
Sounds good - I'll keep them here too cause I suspects that folks will still want to use the readme from Github UI than figuring out how to render the one in stackblitz or following along the unrendered markdown file